lib/engine/game/g_1822.rb

```diff
@@ -50,15 +50,58 @@ class G1822 < Base
 
       attr_accessor :bidding_token_per_player
 
+      def can_run_route?(entity)
+        entity.trains.any? { |t| t.name == 'L' } ? true : super
+      end
+
+      def check_overlap(routes)
+        super
+
+        # Check local train not use the same token more then one time
+        local_token_hex = []
+        routes.each do |route|
+          local_token_hex << route.head[:left].hex.id if route.train.local? && !route.connections.empty?
+        end
+
+        local_token_hex.group_by(&:itself).each do |k, v|
+          raise GameError, "Local train can only use the token on #{k[0]} once." if v.size > 1
+        end
+      end
+
       def entity_can_use_company?(_entity, company)
         # Setting bidding companies owner to bank, make sure the abilities dont show for theese
         company.owner != @bank
       end
 
+      # TODO: Make include with 1817
+      def format_currency(val)
+        # On dividends per share can be a float
+        # But don't show decimal points on all
+        return format('$%.1<val>f', val: val) if val.is_a?(Float) && (val % 1 != 0)
```

Comment:
can we use the other syntax ? '$%.1 ' % val

Reply:
If i try the syntax above i get an RuboCop message:

RuboCop: Favor `format` over `String#%`. [Style/FormatString]